tests/integration/gateway/03-vrrp-ipv6.yml

```diff
@@ -0,0 +1,114 @@
+---
+message: |
+  This lab tests IPv6 VRRP configuration. It checks whether the
+  device-under-test (DUT) forwards traffic sent to the VRRP IP/MAC address,
+  whether it has correct VRRP priority, and whether the preemption mechanism
+  works.
+
+module: [ gateway ]
+
+groups:
+  probes:
+    device: eos
+    provider: clab
+    members: [ r2 ]
+  hosts:
+    device: linux
+    provider: clab
+    members: [ h1, h2 ]
+
+gateway.protocol: vrrp
+gateway.id: 42
+gateway.vrrp.group: 17
+
+nodes:
+  dut:
+  r2:
+  h1:
+  h2:
+
+links:
+- h1:
+  dut:
+    gateway.vrrp.priority: 30
+  r2:
+    gateway.vrrp.priority: 20
+  gateway: True
+  prefix.ipv6: 2001:db8:bad:cafe::/64
+- dut:
+  r2:
+  h2:
+  prefix.ipv6: 2002:db8:bad:cafe::/64
+
+validate:
+  r2_eth1_down:
+    description: Remove R2 from the VRRP LAN
+    nodes: [ r2 ]
+    config:
+      template: ifdown
+      variable.ifstate: 'down'
+    pass: R2 has been disconnected from the VRRP LAN
+    stop_on_error: True
+  ra:
+    description: Check RA-generated default route
+    wait: 30
+    wait_msg: Waiting for RA message to generate the default route
+    nodes: [ h1, h2 ]
+    plugin: default6()
```

Comment:
Issue here is multiple default routes

Reply:
Yeah, that's always the issue with multiple IPv6 routers attached to the same subnet. I wrote about that 11 years ago: https://blog.ipspace.net/2014/02/first-hop-load-balancing-in-ipv6/